Apps/release-planner/deployment.md

```diff
@@ -0,0 +1,186 @@
+# Deployment
+
+Follow the steps given in this document to install the Release Planner App in your environment.
+
+### Step by step instruction
+
+1.  On the environment where you intend to import the solution go to
+    “Solutions”.  
+    #### Classic UI
+    
+
+    ![Classic UI - Advanced Settings](media/classicui-advancedsetting.png)
```

Comment:
images are not rendering. Please reformat the images to include "./" (example: ./media/classicui-advancedsettings.png)

Reply:
done

Reply:
This is still not updated, please add the correct format ( **./**media/...)